src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -505,15 +529,16 @@ private function createNewInvoice(Order $order, Cart $cart, $isFreeShipping)
      */
     private function updateExistingInvoice($orderInvoiceId, Cart $cart)
     {
+        $precision = $this->getPrecisionFromCart($cart);
         $invoice = new OrderInvoice($orderInvoiceId);
 
         $invoice->total_paid_tax_excl += Tools::ps_round(
```

Comment:
Something feels off For the order you replaced the `+=` part with a simple `=` I guess it's because previously the cart only contained the new product but now it contains all o them
So it should be the same thing when you update the invoice right?

Reply:
Correct. I fixed it.